test/lib/sstable_utils.cc

```diff
@@ -208,3 +208,35 @@ void compaction_manager_test::deregister_compaction(const sstables::compaction_d
         testlog.error("compaction_manager_test: deregister_compaction uuid={}: task not found", c.compaction_uuid);
     }
 }
+
+shared_sstable verify_mutation(test_env& env, shared_sstable sst, lw_shared_ptr<replica::memtable> mt, bytes key, std::function<void(mutation_opt&)> verify) {
+    auto sstp = make_sstable_containing(std::move(sst), mt);
+    return verify_mutation(env, std::move(sstp), std::move(key), std::move(verify));
+}
+
+shared_sstable verify_mutation(test_env& env, shared_sstable sstp, bytes key, std::function<void(mutation_opt&)> verify) {
```

Comment:
There was `verify_mutation` lambda in this PR (in sstable_compaction_test). They look very much alike and deserve being merged/generalized